### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Checklist

- [ ] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
- [ ] Update documentation if needed.
- [ ] Update CHANGELOG.md.

### CODE REVIEW

File: docker/router/Dockerfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ RUN touch /tmp/build/yarn.lock
 
 # ----- Install dependencies -----
 # Install dependencies exactly as in the yarn.lock file - no updates.
-RUN yarn install
+RUN SKIP_FORGE=1 yarn install
```
Comment (rhlsthrm) [inline docker/router/Dockerfile:39]: Shouldn't need this anymore.

File: docker/router/config.json [hunk 2]
```diff
@@ -1,37 +1,28 @@
...
+      "subgraph": {
+        "runtime": ["https://api.thegraph.com/subgraphs/name/connext/nxtp-amarok-runtime-v0-kovan"],
+        "analytics": [""]
+      },
+      "deployments": {},
+      "assets": [{ "name": "TEST", "address": "0x80dA4efc379E9ab45D2032F9EDf4D4aBc4EF2f9d" }]
     }
-  ]
+  },
+  "mnemonic": "village fiscal include unaware shrug swear whale sea car tilt clinic despair"
```
Comment (rhlsthrm) [inline docker/router/config.json:27]: No mnemonics in config files plz!

Comment (rhlsthrm) [reply docker/router/config.json:27]: Unless it's the "candy maple..." one:

```
candy maple cake sugar pudding cream honey rich smooth crumble sweet treat
```

File: docker/router/config.json [hunk 3]
```diff
@@ -1,37 +1,28 @@
...
-  "swapPools": [
-    {
-      "name": "TEST",
-      "assets": [
-        { "chainId": 1337, "assetId": "0x345cA3e014Aaf5dcA488057592ee47305D9B3e10" },
-        { "chainId": 1338, "assetId": "0x345cA3e014Aaf5dcA488057592ee47305D9B3e10" }
-      ]
+  "chains": {
+    "2000": {
+      "providers": ["https://rinkeby.infura.io/v3/19b854cad0bc4089bffd0c93f23ece9f"],
```
Comment (rhlsthrm) [inline docker/router/config.json:9]: Also, no infura keys in config.

File: packages/deployments/contracts/package.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "test:hardhat": "hardhat test",
     "test:forge": "yarn forge test --gas-report --hh",
     "forge": "forge",
-    "forge:install": "forge install",
+    "postinstall": "test -n \"$SKIP_FORGE\" || forge install",
```
Comment (rhlsthrm) [inline packages/deployments/contracts/package.json:10]: Can remove this postinstall script completely, it's already pushed to `amarok` branch.

File: docker/router/config.json [hunk 5]
```diff
@@ -1,37 +1,28 @@
...
-      "subgraph": "http://172.17.0.1:8010/subgraphs/name/connext/nxtp",
-      "transactionManagerAddress": "0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0"
-    },
-    "1338": {
-      "providers": ["http://172.17.0.1:8546"],
-      "confirmations": 1,
-      "subgraph": "http://172.17.0.1:9010/subgraphs/name/connext/nxtp",
-      "transactionManagerAddress": "0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0"
-    }
+  "logLevel": "debug",
```
Comment (rhlsthrm) [inline docker/router/config.json:2]: I'm not sure this hardcoded config file needs to be in here for just the 2000/3000 domains (rinkeby, kovan). In the old stuff, this config file was used to spin up a fully local integration test environment. At some point, we will want to replicate that, but I think even then we can structure it better. IMO it shouldn't be inside this `docker/router/` directory/

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

**Comment by rhlsthrm** [comment]
OK, fine to keep the conditional `forge install`.

File: docker/router/config.json (hunk 2 above)
Comment (carlomazzaferro) [reply docker/router/config.json:27]: Alright safe, the one there is one that I've gotten somewhere online that was randomly generated, so shouldn't have any risks. But I'll use that one

**Pull request merged**